pkg/controllers/status/work_status_controller.go

```diff
@@ -165,8 +166,8 @@ func generateKey(obj interface{}) (util.QueueKey, error) {
 
 // getClusterNameFromLabel gets cluster name from ownerLabel, if label not exist, means resource is not created by karmada.
 func getClusterNameFromLabel(resource *unstructured.Unstructured) (string, error) {
```

Comment:
Should rename to getClusterNameFromAnnotation? i am just nitpicking...

Reply:
Nice finding! Would you be willing to submit a pr to update it?

Reply:
Kindly ping @yizhang-zen

Reply:
https://github.com/karmada-io/karmada/pull/4800